.github/ISSUE_TEMPLATE/3-general-issue.yaml

```diff
@@ -0,0 +1,22 @@
+name: General issue
+description: Ask for information, doubts, etc...
```

Comment:
Should we try and direct people to the Discussions page for questions?

Reply:
We can talk about this and improve these templates.